processes/third_party_vuln_process.md

```diff
@@ -0,0 +1,86 @@
+This document describes the management of vulnerabilities within the Node.js
+ecosystem. Vulnerabilities in Node.js core are out of this scope.
```

Comment:
Saying that, we will end up collecting vulnerabilities for Front-end packages.

Reply:
Perhaps link here to https://nodejs.org/en/security/ or something like that?

Reply:
the nsp database includes a couple front-end packages, as does snyk's database. There are lots of FE code distributed via npmjs.org. Node "app" developers (or users of those apps when they are insecure) don't necessarily distinguish between a "vuln in js code evaluated by node" and "vuln in js code served by node to a FE, and evaluated by a web browser".  We need to be really clear, though, on what the scope is if the DB managed by the Node Foundation. Is it packages in npmjs.org? Or, is it node javascript packages (no matter where accessed from)? Both?

Reply:
We historically considered anything within npm to be in scope for what to include in our ecosystem initiative.